### PR TITLE
Fix OIDC Test

### PR DESCRIPTION
**Pull request by westin-m**
Change the URL in the config manager test to prevent an html response from coming back, which causes the test to fail

### CODE REVIEW

**Review by jennyf19 — APPROVED** [review]

**Comment by JoshLozensky** [comment]
Is sending traffic to a random uri potentially insecure? Rather than misspelling it could we use something msft owns consistent with other tests in the repo like contoso.com? If that runs into similar issues, why can't we handle the "<" character?

File: test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/ConfigurationManagerTests.cs [hunk 1]
```diff
@@ -327,7 +327,7 @@ public void GetConfiguration()
             }
 
             // get configuration from https address, should throw
-            configManager = new ConfigurationManager<OpenIdConnectConfiguration>("https://someaddress.com", new OpenIdConnectConfigurationRetriever());
+            configManager = new ConfigurationManager<OpenIdConnectConfiguration>("https://someadress.com", new OpenIdConnectConfigurationRetriever());
```
Comment (jennyf19) [inline test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/ConfigurationManagerTests.cs]:
```suggestion
            configManager = new ConfigurationManager<OpenIdConnectConfiguration>("http://127.0.0.1", new OpenIdConnectConfigurationRetriever());
```

Comment (jennyf19) [reply test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/ConfigurationManagerTests.cs]: matches this [PR](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/ec39ddb318b045ea1b0a2bd993356c6726e5a3bd) and resolves Josh's comment too.

Comment (westin-m) [reply test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/ConfigurationManagerTests.cs]: I'll push something that applies this to both parts of the test that refer to someaddress.com

**Comment by westin-m** [comment]
> Is sending traffic to a random uri potentially insecure? Rather than misspelling it could we use something msft owns consistent with other tests in the repo like contoso.com?

We'll use the localhost address instead.
> If that runs into similar issues, why can't we handle the "<" character?

We're only deserializing JSON, not HMTL.

**Review by JoshLozensky — APPROVED** [review]

**Pull request merged**